docs/Releasing.md

```diff
@@ -7,32 +7,47 @@ example:
 Version Bump
 ------------
 
-We want to release `v0.4.0` in about a week, so we choose a release manager from
-the committers. The initial announcement of the upcoming release will be a PR by
-the release manager to the `master` branch with the commit to bump the [VERSION
-file](../VERSION) to `v0.4.0`, and the commit to update the
-[CHANGELOG](#changelog-updates) "Unreleased" section to `v0.4.0`.
+> A note on naming: please prepend the `v` for consistency. Tags are `v0.4.0`
+> and branches are `v0.4.x`. We follow [Semantic
+> Versioning](https://semver.org/spec/v2.0.0.html).
+
+We want to release `v0.5.0` in about a week, so we choose a release manager from
+the committers.
+
+The initial announcement of the upcoming release will be a PR by the release
+manager to the `master` branch with the commit to bump the [VERSION
+file](../VERSION) to _next_ pre-release, e.g. `v0.6.x`, and the commit to update
+the [CHANGELOG](#changelog-updates) "Unreleased" section to `v0.6.0` (and change
+the previous to v0.5.0).
 
 Release Branch
 --------------
 
-After the version bump PR is merged to `master`, the release manager creates a
-branch off `master` named `v0.4.x`.
+After this is merged, a release branch named `v0.5.x` is created from the last
+commit in `master` before the version changed to `v0.6.x`. The VERSION file in
+this commit should currently have `v0.5.x` in it, and the first commit to the
+release branch is to change it to `v0.5.0-rc1`.
```

Comment:
Currently, appending "-rc1" to the version number in the VERSION file breaks the build.  More changes will be required to support release candidate versioning.

Reply:
Can you post how it breaks so we can fix it?